src/main/java/com/example/WorldOfAnimals/config/OpenAPIConfig.java

```diff
@@ -0,0 +1,58 @@
+package com.example.WorldOfAnimals.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Configuration
+@ConfigurationProperties(prefix = "openapi")
+public class OpenAPIConfig {
+
+    private String devName;
+
+    private String devEmail;
+
+    private Object appName;
+
+    private String appVersion;
+
+    private String appDescr;
+
+    private List<String> serversUrl;
+
+    private List<String> serversDescr;
+
+    @Bean
+    public OpenAPI myOpenAPI() {
```

Comment:
Please swith this bean to another config

Reply:
Done